src/cordova/android/OutlineAndroidLib/outline/src/main/AndroidManifest.xml

```diff
@@ -2,9 +2,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
Why is this needed?

Reply:
Foreground Service, which is `VpnTunnelService` in OutlineAndroidLib, needs this permission with targetSDK 33.

Here is the reference: https://developer.android.com/develop/ui/views/notifications/notification-permission